docs/reference/remap.cue

```diff
@@ -30,7 +30,7 @@ remap: {
 
 			arguments: [...#Argument] // Allow for empty list
 			return: [#RemapReturnTypes, ...#RemapReturnTypes]
-			category:    "coerce" | "parse" | "text" | "hash" | "event"
+			category:    "coerce" | "parse" | "text" | "hash" | "event" | "ip"
```

Comment:
minor nit: do we want a whole separate category for these functions?

If so, maybe `networking` would be a more general alternative?